src/components/Reports/MPGAIncomeExpensesReport/Charts/SummaryBarChart.tsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Bar,
+  BarChart,
+  Cell,
+  LabelList,
+  ResponsiveContainer,
+  XAxis,
+  YAxis,
+} from 'recharts';
+
+interface SummaryBarChartProps {
+  incomeTotal: number | undefined;
+  expensesTotal: number | undefined;
+  aspect: number;
+  width: number;
+}
+
+const chartColors = ['#05699B', '#00C0D8'];
+
+export const SummaryBarChart: React.FC<SummaryBarChartProps> = ({
+  incomeTotal,
+  expensesTotal,
+  aspect,
```

Comment:
[Reuse] Should use existing `currencyFormat` utility from `src/lib/intlFormat.ts` instead of hardcoded `toLocaleString`. The utility provides proper error handling and fallbacks. Replace all currency formatting with: `currencyFormat(value, currency || 'USD', locale)`